bluesky/tests/test_flyer.py

```diff
@@ -370,3 +371,50 @@ def read_configuration(self):
     @call_counter
     def describe_configuration(self):
         return super().describe_configuration()
+
+
+def test_describe_config_optional(RE):
+    class Simple:
+        """A trivial flyer that omits the configuration methods"""
+
+        name = "simple_flyer"
+        parent = None
+
+        def kickoff(self):
+            return NullStatus()
+
+        def describe_collect(self):
+            return {"stream_name": {}}
+
+        def complete(self):
+            return NullStatus()
+
+        def collect(self):
+            for i in range(100):
+                yield {"data": {}, "timestamps": {}, "time": i, "seq_num": i}
+
+        def stop(self, *, success=False):
+            pass
+
+    d = DocCollector()
+
+    flyer = Simple()
+
+    RE(
+        [
+            Msg("open_run"),
+            Msg("kickoff", flyer, group="foo"),
+            Msg("wait", group="foo"),
+            Msg("complete", flyer, group="bar"),
+            Msg("wait", group="bar"),
+            Msg("collect", flyer),
+            Msg("close_run"),
+        ],
+        d.insert,
```

Comment:
Why not use `bp.fly` here?

Reply:
With snark: because this was a few lines above, easy to copy-paste, and I did not remember the name of `bp.fly` ;)

More seriously: The important thing from the point of view of the RE is the sequence of messages so by passing it just the sequence it needs here it is very clear exactly what sequence of messages the test thinks it is testing and insulates this test from any possible changes to bp.fly.